pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -91,17 +93,23 @@ module.exports = {
     diagnosticsButton: '[data-qa="diagnostics-button"]',
     diagnosticsLabel: '[data-qa="diagnostics-label"]',
     downloadLogsButton: '//a[@class="ant-btn" and @href="/logs.zip"]',
+    diagnosticsInfo: '//div[@data-qa="diagnostics-label"]/div/div',
     iframe: '//div[@class="panel-content"]//iframe',
     metricsResolutionButton: '[data-qa="metrics-resolution-button"]',
     metricsResolution: '//div[@data-qa="metrics-resolution-radio-button-group"]/label[text()="',
     metricsResolutionLabel: '[data-qa="metrics-resolution-label"]',
     metricsResolutionRadio: '[data-qa="metrics-resolution-radio-button-group"]',
+    loginButton: '[data-qa="sign-in-submit-button"]',
```

Comment:
please change it to '$sign-in-submit-button'

Reply:
please do the same for all 'data-qa' selectors

Reply:
@yurkovychv Could you specify the exact format of this? :) I tried it but everything started failing :) You mean '${sign-in-submit-button}'?

Reply:
@IvanaBrecherova no, actually it should be `loginButton: '$sign-in-submit-button'`

Reply:
Now I understand. Thanks! 👍